enumerables.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    i = 0
+    while i < size
+      yield(self[i])
+      i += 1 
+    end
+    self
+  end
+
+  def my_each_whit_index
```

Comment:
You should an enumerable, if block is not given after defining the method. Also check the spelling of index- in your method name.